medicines/search-client/src/lib.rs

```diff
@@ -20,8 +20,8 @@ pub struct AzureSearchClient {
 }
 
 pub struct AzurePagination {
-    pub result_count: u32,
-    pub offset: u32,
+    pub result_count: i32,
+    pub offset: i32,
```

Comment:
Don't these both want to stay as `u32` so they can't ever be negative?

Reply:
Juniper had problems with dealing with `u32`s.

Reply:
Ah, fair enough